src/allmydata/test/common.py

```diff
@@ -1,4 +1,7 @@
+from __future__ import absolute_import
+from __future__ import division
 from __future__ import print_function
+from __future__ import unicode_literals
```

Comment:
If you're going to do this you need to audit the whole module. E.g. on line 424 there's a string that should be a byte string now, probably other cases. I recommend not doing future imports unless you're explicitly actually porting the module.

This is the likely cause of the at least some of the Python 2 failures.

Reply:
Makes sense. Done in 3ae927eab951fbb7f28cc85e8c2247f906a9674c. C'mon, CI! No whammies! 🎲  🎲